components/sceneutil/riggeometry.cpp

```diff
@@ -350,16 +352,22 @@ void RigGeometry::setInfluenceMap(osg::ref_ptr<InfluenceMap> influenceMap)
 
         for (auto& weightPair: bi.mWeights)
         {
-            std::vector<BoneWeight>& vec = vertex2BoneMap[weightPair.first];
-
-            vec.emplace_back(std::make_pair(boneName, bi.mInvBindMatrix), weightPair.second);
+            std::pair<float, std::vector<BoneWeight> >& vec = vertex2BoneMap[weightPair.first];
```

Comment:
You've made a typedef for this a few lines above, and then not used it. Surely it's best to be consistent?

Reply:
arf forgot this one